pkg/agent/controller/noderoute/node_route_controller_unix.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// +build linux
+
+package noderoute
+
+import (
+	"fmt"
+	"net"
+
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/klog"
+)
+
+func (c *Controller) reconcileIPTables() error {
+	nodes, err := c.nodeLister.List(labels.Everything())
+	if err != nil {
+		return fmt.Errorf("error when listing Nodes: %v", err)
+	}
+
+	// collect CIDRs from each known Node, and restore its iptables rule.
+	for _, node := range nodes {
+		_, podCIDR, err := net.ParseCIDR(node.Spec.PodCIDR)
+		if err != nil {
+			klog.Errorf("Failed to parse Pod CIDR for Node %s: %v", node.Name, err)
+			continue
+		}
+		peerNodeIP, err := GetNodeAddr(node)
+		if err != nil {
+			klog.Errorf("Failed to get IP for Node %s: %v", node.Name, err)
+			continue
+		}
+		err = c.iptablesClient.AddPeerCIDR(podCIDR, peerNodeIP)
+		if err != nil {
+			klog.Errorf("Failed to update iptables for node %s: %v", node.Name, err)
+			continue
+		}
+	}
+	if err := c.iptablesClient.Reconcile(); err != nil {
+		return fmt.Errorf("iptables reconcile: %v", err)
+	}
+	return nil
+}
+
+// allowTrafficToPeerCIDR adds iptables rules relevant to peerPodCIDR.
+func (c *Controller) allowTrafficToPeerCIDR(peerPodCIDR *net.IPNet, peerNodeIP net.IP) error {
```

Comment:
How about name it allowNodeToPeerCIDRTraffic?

Another question - do you think we should change Linux to SNAT in OVS too, so to unify Windows and Linux? @tnqn